reconplogger_tests.py

```diff
@@ -119,6 +119,9 @@ def test_flask_app_logger_setup(self):
         app = Flask(__name__)
         reconplogger.flask_app_logger_setup(
             env_prefix=env_prefix, flask_app=app)
+        assert app.logger.filters
+        assert app.before_request_funcs
+        assert app.after_request_funcs
```

Comment:
Would be good to add two new unit tests:
1. Do a request to a flask app **without** a correlation id in the header. Capture the logs and make sure that a correlation id is present.
2. Do a request to a flask app **with** a correlation id in the header. Capture the logs and make sure that a correlation id is present and that it is equal to the one provided in the header of the request.